mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -1943,9 +1943,24 @@ static void emitPrintTensor(OpBuilder &b, Value var) {
   }
 }
 
+static void checkRandomInputsE2E() {
+  if (randomSeed != "none" && randomSeed != "fixed" &&
+      randomDataType == "float") {
+    int min = randMin.getValue();
+    int max = randMax.getValue();
+    if ((-1 <= min && min < 1) || (-1 < max && max <= 1)) {
```

Comment:
How about `(min < 0 && max > 0)`? I think in general as long as there is a chance to cancel out each other, there's going to be denorm behavior.